crates/ruff_linter/src/checkers/logical_lines.rs

```diff
@@ -24,7 +24,7 @@ pub(crate) fn expand_indent(line: &str, indent_width: IndentWidth) -> usize {
     let tab_size = indent_width.as_usize();
     for c in line.bytes() {
         match c {
-            b'\t' => indent = (indent / tab_size) * tab_size + tab_size,
```

Comment:
Wait sorry, _is_ this the same? What if `indent` is 4 and `tab_size` is 3?

`indent += tab_size` would yield 7, but `indent = (indent / tab_size) * tab_size + tab_size` would yield 6, right?

Reply:
I.e., is the goal here perhaps that `indent` ends up as a multiple of `tab_size`?

Reply:
Is the deal here that there's implicit integer rounding?

Reply:
If so, can we add a comment explaining the deal? @charliermarsh